packages/console/src/assets/docs/tutorial/integrate-sdk/express.mdx

```diff
@@ -0,0 +1,384 @@
+import UriInputField from '@mdx/components/UriInputField';
+import Step from '@mdx/components/Step';
+import Tabs from '@mdx/components/Tabs';
+import TabItem from '@mdx/components/TabItem';
+import Alert from '@/components/Alert';
+
+<Step
+  title="Add Logto SDK as a dependency"
+  subtitle="Please select your favorite package manager"
+  index={0}
+  activeIndex={props.activeStepIndex}
+  onButtonClick={() => props.onNext(1)}
+>
+
+The express demo app will need 4 dependencies:
+
+1. **@logto/js**: Logto's core SDK for JavaScript.
+2. **node-fetch**: Minimal code for a `window.fetch` compatible API on Node.js runtime.
+3. **express-session**: A session middleware, we'll use the session to store user tokens.
+4. **js-base64**: Yet another Base64 transcoder.
+
+<Tabs>
+<TabItem value="npm" label="npm">
+
+```bash
+npm i @logto/js node-fetch@v2 express-session js-base64
+```
+
+</TabItem>
+<TabItem value="yarn" label="Yarn">
+
+```bash
+yarn add @logto/js node-fetch@v2 express-session js-base64
+```
+
+</TabItem>
+<TabItem value="pnpm" label="pnpm">
+
+```bash
+pnpm add @logto/js node-fetch@v2 express-session js-base64
+```
+
+</TabItem>
+</Tabs>
+</Step>
+
+<Step
+  title="Use session"
+  subtitle="1 step"
+  index={1}
+  activeIndex={props.activeStepIndex}
+  onButtonClick={() => props.onNext(2)}
+>
+
+When users are signed in, they will get a set of tokens (Access Token, ID Token, Refresh Token) and interaction data, and the session is an excellent place to store them.
+
+We have installed [express-session](https://github.com/expressjs/session) in the previous step, now setup for it by adding some code to `app.js`:
```

Comment:
nit: maybe no `for it`

Reply:
`adding some code` sounds informal

Reply:
Yeah, you are right. This is directly migrated from docs repo and I didn't fully check the content, thanks!
How about 
```
We have installed express-session in the previous step, so now simply add the following code to set it up.
```

Reply:
Cool